app/assets/stylesheets/control.scss

```diff
@@ -0,0 +1,4 @@
+.myStyle {
+  color: #aaaaaa;
```

Comment:
Color literals like `#aaaaaa` should only be used in variable declarations; they should be referred to via variable everywhere else.